db/revision_test.go

```diff
@@ -0,0 +1,38 @@
+package db
+
+import (
+	"testing"
+	"log"
+)
+
+func TestParseRevID(t *testing.T) {
+
+	var generation int
+	var digest string
+
```

Comment:
A revision's digest is an MD5 checksum in hex (as string), so should only have the characters `[a-fA-F0-9]`.

Our current ParseRevID doesn't enforce that, but it's probably reasonable to limit the tests to those values (i.e. an alternate implementation of ParseRevID doesn't actually need to handle something like 2222-lj-lkjl).